### PR TITLE
Get edge

### PR DESCRIPTION
**Pull request by lhchandler4**
- [ X] Top-level README “Table of Contents” is updated
 - [ X] Feature tasks for this challenge are completed
 - [ X] Unit tests written and passing
     - [ X] “Happy Path” - Expected outcome
     - [ X] Expected failure
     - [ X] Edge Case (if applicable/obvious)
 - [ X] README for this challenge is complete
     - [ X] Summary, Description, Approach & Efficiency, Solution
     - [ X] Link to code
     - [ X] Picture of whiteboard

### CODE REVIEW

File: challenges/src/main/java/Java/GetEdge/GetEdge.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package Java.GetEdge;
+import Java.Graph.Node;
+import Java.Graph.Edge;
+import Java.Graph.Graph;
+import java.util.HashSet;
+
+public class GetEdge<T> {
+
+        boolean directPossible = false;
```
Comment (evanslaton) [inline challenges/src/main/java/Java/GetEdge/GetEdge.java:9]: These two lines are improperly indented!

File: challenges/src/main/java/Java/GetEdge/GetEdge.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        int cost = 0;
+
+    public GetEdge(){}
```
Comment (evanslaton) [inline challenges/src/main/java/Java/GetEdge/GetEdge.java:12]: You don't need a no-args constructor unless you are using Spring and interacting with a database

File: challenges/src/main/java/Java/GetEdge/GetEdge.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    public GetEdge(boolean output, int cost){
+            this.directPossible = output;
+            this.cost = cost;
+        }
+        public GetEdge directRoute(Graph input, String[] inputArr){
```
Comment (evanslaton) [inline challenges/src/main/java/Java/GetEdge/GetEdge.java:18]: Everything below this line is improperly indented

File: challenges/src/main/java/Java/GetEdge/GetEdge.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            HashSet<Node> n = input.getNodes();
+            Node current = null;
+            int cost = 0;
```
Comment (evanslaton) [inline challenges/src/main/java/Java/GetEdge/GetEdge.java:21]: The class already has a property called `cost`, this might confuse anyone reading your code (including future you) because they don't immediately know which one you are meaning to use
